### DISCUSSION TITLE
NX16+ upgrade is discontinuing usage of the nx generate @nrwl/workspace:library

### DISCUSSION THREAD

**Discussion by rostag** [wgd3/full-stack-todo#22] [General]
Wallace, thanks for the good article on adding feature library (https://thefullstack.engineer/full-stack-development-series-part-2-creating-a-rest-api-with-nestjs/#/portal/signup)

One note: the first generation command mentioned there needs rewriting after rescoping of the @nrwl packages to @nx (mentioned here: https://nx.dev/recipes/other/rescope).

So this: 

```npx nx generate @nrwl/workspace:library domain --directory=shared --importPath=@fst/shared/domain --skipBabelrc --standaloneConfig --tags=scope:shared,type:domain```

Would become this:

```npx nx generate @nx/js:library domain --directory=shared --importPath=@mtm/shared/domain --tags=scope:shared,type:domain```

Warmest regards from Ukraine :)
R.

**Comment by wgd3** [comment]
@rostag thanks for reading the blog and bringing this to my attention! I've updated that specific post and given you credit in the Updates section ;) 

I'll go through the rest of my posts in this series and get them updated as well.

**Reply by rostag** [reply]
@wgd3 thanks for taking care! Imagine that – I came here by link from that updated blog ;) 

The thing is, your writing is not just useful, but also reusable. I mean, after I went all the way through 6 parts of your tutorial, I couldn't resist to start another feature lib / service / DTO for my other project, so went back to your text just to refresh the knowledge. After I read Part 2 all the way down, I saw your mention of my feedback from previous reading. Thanks for that :)
